js/ui/grid_core/ui.grid_core.error_handling.js

```diff
@@ -17,55 +17,69 @@ var ErrorHandlingController = modules.ViewController.inherit({
         that._rowsView = that.getView("rowsView");
     },
 
-    _createErrorRow: function(message, $tableElements) {
+    _createErrorMessage: function(message, $tableElements) {
         var that = this,
-            $errorRow = $("<tr />").addClass(ERROR_ROW_CLASS),
-            $errorMessage = $("<div/>").addClass(ERROR_MESSAGE_CLASS).text(message),
+            $errorRow,
+            $closeButton,
```

Comment:
I think it would be better to define these variables inside the `if($tableElements)` block